status-im/status-react
# [BUG #2881] Migrated wallet main screen to latest mockups

fixes #2881

### Summary:

Adapt wallet main screen to latest mockups

### Steps to test:
- Open Status
- Navigate wallet tab
- Verify it matches mockups

EDIT: mocks to check with: https://app.zeplin.io/project/5a55d587ba304eed4d2c942f/screen/5a589a0fccc89d66c86d6be8
status: ready
